src/main/java/org/jenkinsci/plugins/authorizeproject/ConfigurationPermissionEnforcer.java

```diff
@@ -47,16 +48,28 @@ public String getDisplayName() {
         public JobProperty<?> newInstance(StaplerRequest req, JSONObject formData) throws FormException {
             Job<?,?> job = req.findAncestorObject(Job.class);
             if (job != null) {
-                AuthorizeProjectProperty property = job.getProperty(AuthorizeProjectProperty.class);
-                if (property != null && ProjectQueueItemAuthenticator.isConfigured()) {
-                    AuthorizeProjectStrategy strategy = property.getStrategy();
-                    if (strategy != null) {
-                        strategy.checkConfigurePermission(job);
-                    }
-                }
+                checkConfigurePermission(job);
```

Comment:
Extract to `checkConfigurePermission` to avoid deep indentations and add `ADMINISTER` check.